Gemfile

```diff
@@ -16,22 +16,23 @@ gemspec
 gem "defra_ruby_template"
 
 # Use MongoDB as the database
-gem "mongoid", "~> 7.5"
+gem "mongoid"
```

Comment:
It might be best to re-pin mongoid and devise (at least) to their new/updated major versions.